resources/public/api/conf/1.0/retrieveCodingOutUnderpayments.yaml

```diff
@@ -0,0 +1,102 @@
+
+description: |
+  This endpoint allows the developer to retrieve HMRC held coded out underpayments and coded out debt amounts or user submitted amended coded out underpayments and coded out debt amounts for a previous tax year and given NINO.
+  
+  
+  ### Test data
+    <p>Scenario simulations using <code>Gov-Test-Scenario</code> headers is only available in the sandbox environment.</p>
+    <table>
+        <thead>
+            <tr>
+                <th>Header Value (Gov-Test-Scenario)</th>
+                <th>Scenario</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td><p>N/A - DEFAULT</p></td>
+                <td><p>Simulate a successful response with a self-assessment accounts.</p></td>
+            <tr>
+                <td><p>N/A - DEFAULT <br> using ?source=latest </p></td>
+                <td><p> Simulates a successful response with latest values.</p></td>
+            <tr>
+                <td><p>N/A - DEFAULT <br> using ?source=hmrcHeld </p></td>
+                <td><p>Simulates a successful response with HMRC-held values.</p></td>
+            <tr>
+                <td><p>N/A - DEFAULT <br> using ?source=user </p></td>
+                <td><p>Simulates a successful response with user submitted values.</p></td>
+            <tr>
+              <td><p>NOT_FOUND</p></td>
+              <td><p>Simulates the scenario where no data is found.</p></td>
+            </tr>
+        </tbody>
+    </table>
+
+tags:
+  - Coding Out Underpayments and Debts
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
```

Comment:
The taxYear description here isn't exactly right - it says the minimum taxYear in sandbox is 2021-22, but the raml for this endpoint says it's 2019-20. 
<img width="639" alt="image" src="https://user-images.githubusercontent.com/106095575/216292710-6e9eacb2-91c9-43d4-b988-f42d5189371d.png">